docs/vta/install.md

```diff
@@ -66,6 +66,60 @@ cp vta/config/vta_config.json vta_config.json
 make vta
 ```
 
+## VTA TSIM Installation
+
+*TSIM* is a cycle accurate hardware simulation environment for VTA accelerators that can be invoked and managed directly from TVM.
+This simulation environment can be used in both OSX and Linux.
+There are two dependencies required to make *TSIM* works: [Verilator](https://www.veripool.org/wiki/verilator) and [sbt](https://www.scala-sbt.org/) for accelerators designed in [Chisel3](https://github.com/freechipsproject/chisel3)
+
+### OSX Dependencies
+
+Install `sbt` and `verilator` using [Homebrew](https://brew.sh/)
+
+```bash
+brew install verilator sbt
+```
+
+### Linux Dependencies
+
+Add `sbt` to package manager (Ubuntu).
+
+```bash
+echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
+sudo apt-get update
+```
+
+Install `sbt` and `verilator`.
+
+```bash
+sudo apt install verilator sbt
+```
+
+### Setup in TVM
+
+1. Install `verilator` and `sbt` as described above
+2. Build tvm and enable VTA TSIM by turning on the switch `USE_VTA_TSIM` in config.cmake
+3. Build tvm
+
+### How to run VTA TSIM examples
+
+There are two sample VTA accelerators (add-by-one) designed in Chisel3 and Verilog to show how *TSIM* works.
+These examples are located at `<tvm-root>/vta/apps/tsim_example`
+
+* Instructions
+    * Open `<tvm-root>/vta/apps/tsim_example/python/tsim/config.json`
+    * Change `TARGET` from `verilog` to `chisel`, depending on what language backend you would like to test
+    * Go to `tvm/vta/apps/tsim`
+    * Run `make`
+
+* Some pointers
+    * Build cmake script for driver `<tvm-root>/vta/apps/tsim_example/cmake/modules/driver.cmake`
+    * Build cmake script for tsim `<tvm-root>/vta/apps/tsim_example/cmake/modules/tsim.cmake`
+    * Software driver that handles the VTA accelerator `<tvm-root>/vta/apps/tsim_example/src/driver.cc`
+    * VTA add-by-one accelerator (Verilog) `<tvm-root>/vta/apps/tsim_example/hardware/verilog`
+    * VTA add-by-one accelerator (Chisel) `<tvm-root>/vta/apps/tsim_example/hardware/chisel`
+
 ## VTA Pynq-Based Test Setup
 
 This second guide extends the *VTA Simulator Installation* guide above to run FPGA hardware tests of the complete TVM and VTA software-hardware stack.
```

Comment:
We'll need to change the ordering of the guides; I think that until TSIM can replace the software simulation tests, we should keep it last, after FPGA toolchain installation. Once we have the capability to run full workloads, it should go in second position.

Also perhaps we should rethink the titles here? Currently we have 1. Simulator installation, 2. Hardware test setup, 3. FPGA toolchain. It might be appropriate to change "Simulator" to "Emulator", and rename "VTA TSIM Installation" to "TSIM Cycle-Accurate Simulator Installation"

Reply:
Done.